docker/deployment/db-deployment.yaml

```diff
@@ -23,8 +23,37 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 3Gi
-  storageClassName: weekly-snapshots-retain-4 # provided by LTOps
+      storage: 10Gi
+
+---
+# https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: lf-mongo-data-snapshot-runner
+spec:
+  schedule: "@weekly" # https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: snapshot-runner
+            image: docker.dallas.languagetechnology.org/ba-kb-snapshot
+            command:
+            - /bin/sh
+            - -c
+            - |- 
+              cat <<EOF | kubectl apply -f - 
```

Comment:
Wow, a snapshot container that does the backup, run from a `kubectl` run from a shell script, run from a cronjob defined in a kubernetes YAML.  This is complex!

Reply:
yes, I agree...too complex imo but this is a POC and things like this are what it's all about.  The syntax is pretty gross but if the only thing teams needed to change were the names, we could parameterize this in a way that teams could just send in those two custom names and the underlying container would hide all this ugliness.